bower.json

```diff
@@ -9,23 +9,28 @@
     "polymer",
     "progress"
   ],
-  "main": "paper-progress.html",
+  "main": "paper-linear-progress.html",
   "repository": {
     "type": "git",
-    "url": "git://github.com/PolymerElements/paper-progress.git"
+    "url": "git://github.com/PolymerElements/paper-linear-progress.git"
   },
   "dependencies": {
     "polymer": "Polymer/polymer#^1.1.0",
-    "iron-range-behavior": "PolymerElements/iron-range-behavior#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "iron-flex-layout": "PolymerElements/iron-flex-layout#^1.0.0"
+    "neon-animation": "PolymerElements/neon-animation#^1.0.0"
   },
   "devDependencies": {
     "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
     "test-fixture": "PolymerElements/test-fixture#^1.0.0",
-    "paper-button": "PolymerElements/paper-button#^1.0.0",
     "web-component-tester": "*",
```

Comment:
This is orthogonal to this change, but could you please set this dependency to reflect the following configuration:
`"web-component-tester": "polymer/web-component-tester#^3.4.0"`.